specification/transform/get_transform_stats/types.ts

```diff
@@ -100,9 +101,10 @@ export class CheckpointStats {
 
 export class Checkpointing {
   changes_last_detected_at?: long
-  changes_last_detected_at_date_time?: DateTime
+  changes_last_detected_at_string?: DateTime
   last: CheckpointStats
   next?: CheckpointStats
   operations_behind?: long
   last_search_time?: long
+  last_search_time_string?: DateTime
 }
```

Comment:
Defined in https://github.com/elastic/elasticsearch/blob/6503c1b94bf077a9fc152019932a79c4e4f42d40/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/transforms/TransformCheckpointingInfo.java#L209-L230